lib/compat/wordpress-6.5/block-bindings/sources/post-meta.php

```diff
@@ -13,8 +13,7 @@ function gutenberg_block_bindings_post_meta_callback( $source_attrs ) {
 	if ( isset( $source_attrs['postId'] ) ) {
 		$post_id = $source_attrs['postId'];
 	} else {
-		// I tried using $block_instance->context['postId'] but it wasn't available in the image block.
-		$post_id = get_the_ID();
+		$post_id = isset( $block_instance->context['postId'] ) ? $block_instance->context['postId'] : get_the_ID();
```

Comment:
In which case the fallback is needed?

Reply:
Probably it should never be needed 😄 I can remove that

Reply:
I'm still not sure how `$source_attrs['postId']` would be set here. Most of the blocks inherit the post id from the context so `get_the_ID()` would also be set to the same value as the parent block. Maybe, in the case where there is no parent block that sets the context, `get_the_ID()` is the way to go.

Reply:
> Maybe, in the case where there is no parent block that sets the context, get_the_ID() is the way to go.

In my mind there's always a parent context, because if I'm not wrong we probably set `get_the_ID()` as the default context top level no?

Reply:
For `$source_attrs['postId']` I agree, it seems useless to me and we should just remove that condition.

Reply:
We need some tests to cover all those scenarios. Post meta is a very difficult one to start with.